src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -95,6 +95,13 @@ public void testContent() throws Exception {
     }
     
     @Test
+    public void testSessionWithNoSections() {
+        resultsPage = loginToInstructorFeedbackResultsPageWithViewType("CFResultsUiT.instr",
+                "Session with no sections", true, "question");
+        assertEquals("", resultsPage.getStatus());
```

Comment:
I don't think this deserves its own `@Test` segment. Include this as part of `testHtmlContent`?